dapps/evancore.vue.libs/src/components/forms/form/form.vue

```diff
@@ -57,33 +57,42 @@
       </form>
     </div>
     <template v-if="(editMode || isLoading) && !onlyForm">
-      <a
-        class="text-muted mt-4 mb-3 d-inline-block"
-        href="https://evannetwork.github.io/docs/other/glossary.html#e"
-        target="_blank"
-        rel="noopener noreferrer"
-        v-if="!isLoading">
-        <i class="mdi mdi-information-outline mr-2" />
-        {{ '_evan.transaction_costs_hint' | translate }}
-      </a>
-      <br />
-      <evan-button type="primary" class="mr-3"
-        :disabled="isLoading || disabled || (form && !form.isValid)"
-        :isLoading="isLoading"
-        :label="'_evan.save' | translate"
-        @click="save"
-      />
-      <evan-button type="secondary"
-        v-if="!isLoading"
-        @click="cancel"
-        :label="'_evan.cancel' | translate"
-      />
+      <div class="d-flex justify-content-end">
+        <div>
+          <a
+            class="text-muted mb-3 d-block text-right"
+            href="https://evannetwork.github.io/docs/other/glossary.html#e"
+            target="_blank"
+            rel="noopener noreferrer"
+            v-if="!isLoading"
+          >
+            <i class="mdi mdi-information-outline mr-2" />
+            {{ '_evan.transaction_costs_hint' | translate }}
+          </a>
+          <div>
```

Comment:
```suggestion
          <div class="d-flex justify-content-end">
```

Text of the link will have a greater width than the buttons, so a space will be left

![image](https://user-images.githubusercontent.com/7114819/66558034-2c539980-eb53-11e9-8609-a09181a7335b.png)